### ISSUE TITLE
Milk substitute/ Milk analog / Imitation milk

### ISSUE THREAD

**Issue by ddooley** [FoodOntology/foodon#218]
FoodOn currently has three classes of milk:
Milk substitute
Milk analog
Imitation milk

Some turf war about semantics of what "milk" can refer to has occurred.  The FoodOn crew has leaned towards "milk analog" to establish a connection between bovine milk and plant or nut milk.  But is there a role for the terms "milk substitute" and "imitation milk" in terms of food composition?  Is there a need for indicating vegan substitution, or other taste or functionality objectives. In general, is there a better hierarchy, and can some of these terms be unified?

<img width="379" alt="image" src="https://user-images.githubusercontent.com/4000582/184027644-1e68d08a-8b3b-462b-b4e0-206543ac6313.png">

**Comment by maweber-bia** [comment]
hi,

this is actually a turf war but I think that the 3 terms have quite the same meaning (you may rather distinguish between the liquid or dry form)

However, I am wandering about the parent term "milk based product analog": is there milk in the products or only milk substitute/analog/imitation?

and second, the same questioning applies to "dairy product analog" vs " imitation dairy product"

**Comment by maweber-bia** [comment]
we may also add "milk alternative" as a synonym (it will also be a term "meat alternative" synonym for "meat analog" but that's another branch of course, it is for consistency purpose in case of vegan substitution)

**Comment by cmrn-rhi** [comment]
> However, I am wandering about the parent term "milk based product analog": is there milk in the products or only milk substitute/analog/imitation?

I agree that this can be confusing as to whether it is it a product analog made with milk or the product analog of milk.

**Comment by ddooley** [comment]
I like the idea of having a "milk analog" (syn. milk substitute, imitation milk) but it having only generic subclasses like "imitation lowfat milk" (so we move existing "soy milk substitute (hypoallergenic)" class out of there.  We leave it to Rhiannon's https://github.com/FoodOntology/foodon/issues/212 plant milk hierarchy to hold milks by their source derivation.

**Comment by cmrn-rhi** [comment]
## 2022.11.10 - FoodOn Curation Meeting:

Want to avoid conveying a specific "state" in the upper level food product branch.
Need to remove the UBERON "milk" from the food product branch and just leave it under "bodily fluids"

- "mammalian milk food product" (not state specific) _is derived from_ "milk" [UBERON Term] (which is a liquid)
  - mammalian milk (liquid)
  - mammalian milk (powdered)
  - mammalian milk by organism
    - cow milk
      - cow milk (liquid)
      - cow milk (powdered)

Want to do the same thing for non-plant milks, making clear in the annotations that the use of "dairy" in "non-dairy" applies to mammalian milk products not just standard cattle.

- food product
  - non-dairy milk product
    - plant milk product
      - nut milk product
      
_P.S. Sorry for all the edits email folk - trying to make things more clear._

**Comment by cmrn-rhi** [comment]
Sorry, not sure if these meeting notes are that relevant here. We decided in meeting clearing this up would help us move forward with the substitute/analog/imitation discussion.

**Comment by ddooley** [comment]
Looks good!  My only thought is that non-dairy milk product could be placed under "food by quality" since the general category of "milk" is a substance with loosely a certain quality to it.  It could go under "food product by organism", if we allow that cellular (e.g. yeast) agriculture food products can fit there too.  Hmm.

**Comment by amwindsor** [comment]
US FDA is in the comment stage for guidance on plant-based milk alternatives (PBMA), but the guidance has more to do with nutrition labeling if the word milk is used.  [https://www.fda.gov/regulatory-information/search-fda-guidance-documents/draft-guidance-industry-labeling-plant-based-milk-alternatives-and-voluntary-nutrient-statements]. The gist seems to be that "milk" can be used for PBMA's. Some highlights:
Common or usual names have been established by common usage for some plant-based
milk alternatives. Among these names are “soy milk” and “almond milk” and others that
qualify the term “milk” with the plant source of the food. Names that qualify the terms
“beverage” or “drink” with the plant source of the food are used less frequently, but also
appear to be in common usage. These names include “soy beverage” and “almond
beverage.”The name may be a single word (i.e.,“soymilk”), multiple words (e.g., “soy milk”), or hyphenated (e.g., “soy-milk”).

**Comment by maweber-bia** [comment]
In order to avoid confusion with plant-based beverages, the European Court of Justice ruled on June 14, 2017 that the use of the names "milk" and "cheese" is prohibited when associated with plant-based products. However, there is a list of exceptions provided by the regulation.
The list of exceptions, depending on the country, can be found in the European Decision 2010/791/EU of 20 December 2010 (Annex 1). [https://eur-lex.europa.eu/legal-content/FR/ALL/?uri=CELEX:32010D0791]

For example, there is an exception in France that allows the use of the term "almond milk" (lait d'amande) and "coconut milk" (lait de coco) due to their traditional uses.

**Comment by ddooley** [comment]
I thought I'd try this FoodOn position explanation, for feedback, and to enter into FoodOn website content.  

I can see at least 6 competing camps in this kerfuffle vying for an encompassing or exclusive milk flag.  
- The epidemiologists who want to know if they're dealing with an animal or plant based milk infectious disease outbreak.
- Industry product marketing managers (with consumers in mind) who want the term to be exclusive to mammalian milk, or want it to be inclusive of plant products.
- Industrialists, economists and environmentalists trying to figure out the momentum of plant based or yeast-based milk tech/adoption.
- People who are looking for products with a milk-like quality - the organoleptics of it, and motivated by dietary or environmental or ethical reasons.
- Home and production line experiments involving milk sourcing and substition.
- Legislators & Legals responding to one or the other community.

Any objections to how I used the term "milk" above? :o)

FoodOn is serving all these people - we want to provide semantically precise terms for each kind of user, and not to the exclusion of others.  Consequently no user group gets to claim the "milk" flag, but through synonymy and subclass labelling we should be able to provide semantic granularity. 

Now, UBERON did claim the term "[milk](http://purl.obolibrary.org/obo/UBERON_0001913)" to cover "mammalian milk". The proposed hierarchy above basically does "back-room" references to this term, but up front, this hierarchy should work for all our users? We allow "milk" to be qualified by "plant", yeast, etc. to cover substances with a milk-like quality.  This favours common-use semantics, but we'll have precise identifiers and component descriptions that cater to each and every community even if the labels aren't universally favoured.

**Comment by maweber-bia** [comment]
>  This favours common-use semantics, but we'll have precise identifiers and component descriptions that cater to each and every community even if the labels aren't universally favoured.

This is a good point!
but "component descriptions" need further explanation to avoid confusion!

> UBERON did claim the term "[milk](http://purl.obolibrary.org/obo/UBERON_0001913)" to cover "mammalian milk". 

Actually, that is the biological truth ;-)

**Comment by maweber-bia** [comment]
Follow-up from FoodON Hackathon: revise the naming of the classes under the parent "food material analog" for consistency

see also "Is similar to" link for Analog's #238
